exercises/pythagorean-triplet/PythagoreanTripletTest.fs

```diff
@@ -1,36 +1,37 @@
-// This file was created manually and its version is 1.0.0.
+// This file was auto-generated based on version 1.0.0 of the canonical data.
 
 module PythagoreanTripletTest
 
-open Xunit
 open FsUnit.Xunit
+open Xunit
 
 open PythagoreanTriplet
-    
-[<Theory(Skip = "Remove to run test")>]
-[<InlineData(3, 4, 5, true)>]
-[<InlineData(3, 5, 4, true)>]
-[<InlineData(4, 3, 5, true)>]
-[<InlineData(4, 5, 3, true)>]
-[<InlineData(5, 3, 4, true)>]
-[<InlineData(5, 4, 3, true)>]
-[<InlineData(3, 3, 3, false)>]
-[<InlineData(5, 6, 7, false)>]
-let ``Can recognize a valid pythagorean`` (x: int) (y: int) (z: int) (expected: bool) =
-    let actual = triplet x y z
-    isPythagorean actual |> should equal expected
-   
+
+[<Fact>]
+let ``Triplets whose sum is 12`` () =
+    tripletsWithSum 12 |> should equal [triplet 3 4 5]
```

Comment:
What do you think about just using a plain tuple to represent a triplet? So instead of `[triplet 3 4 5]`, it would become `[(3, 4, 5)]`? I think I have a slight preference for that version, even though the old test suite used a `triplet` function.

Reply:
I'm also fine with using plain tuples. I've updated the generator and the example implementation